site/diagrams/sequenceDiagrams/command_control_concept_http.txt

```diff
@@ -0,0 +1,18 @@
+participant Device
+participant HTTP Adapter
+participant AMQP 1.0 Network
+participant Application
+
+Device->HTTP Adapter: 1. send event (hono-ttd=30)
```

Comment:
IMHO it is relevant to point out that we are talking about an HTTP request here to which the device will later get a response. So I think this should be something like
`HTTP request ("POST", "/event", "hono-ttd=30")`

Reply:
Yes, that makes it clearer.